.github/workflows/update-gardenlogin.yaml

```
@@ -22,6 +22,13 @@ jobs:
           files: 'bin/darwin-amd64/gardenlogin_darwin_amd64;bin/linux-amd64/gardenlogin_linux_amd64'
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           release-tag: ${{ steps.build_binary_files.outputs.latest_release_filtered_tag }}
+      - name: Get token for gardener-github-pkg-mngr app
+        id: gardener-github-workflows
+        # getsentry/action-github-app-token@v1, but the exact commit
+        uses: getsentry/action-github-app-token@8f656f2b31c7aaef91ba5cfdbe308d2a4d72e4f4
```

```suggestion
        uses: getsentry/action-github-app-token@8f656f2b31c7aaef91ba5cfdbe308d2a4d72e4f4 # pin@v1
```
this is how I did it in https://github.com/gardener/gardenlogin/pull/7 (resp. this is how the tool 
adds the comments https://www.npmjs.com/package/pin-github-action)